tests/test_cli.py

```diff
@@ -695,3 +694,16 @@ def test_run_report_without_data(tmpdir, runner, devnull):
         'Parsing profile file %s.' % devnull.name,
         'Error: No data to report.']
     assert result.exit_code == 1
+
+
+@pytest.importorskip('setuptools_scm')
```

Comment:
importorskip is NOT a mark, using it like it was will result in strange issues

Reply:
@RonnyPfannschmidt 
Thanks!